cluster-autoscaler/main.go

```diff
@@ -94,7 +94,10 @@ var (
 	scaleDownUnreadyTime = flag.Duration("scale-down-unready-time", 20*time.Minute,
 		"How long an unready node should be unneeded before it is eligible for scale down")
 	scaleDownUtilizationThreshold = flag.Float64("scale-down-utilization-threshold", 0.5,
-		"Node utilization level, defined as sum of requested resources divided by capacity, below which a node can be considered for scale down")
+		"Sum of cpu or memory of all pods running on the node divided by node's corresponding allocatable resource, below which a node can be considered for scale down")
+	scaleDownGpuUtilizationThreshold = flag.Float64("scale-down-gpu-utilization-threshold", 0.5,
```

Comment:
I think the only values that make sense here are 0 (don't restart pods using GPU) or 1 (try to keep maximum GPU utilization). I don't think there is much use-case for 0.5, so I'd prefer a different default. Feels like most people don't want GPU workloads restarted, so I'd go for 0.0.

Reply:
Last time we talked about use case, we think this would be used for inference workloads, training jobs can not be interrupted and they will be labeled. `cluster-autoscaler.kubernetes.io/safe-to-evict: false`

Inference pods can be restarted and behavior should be close to general service pod. 0 here means no gpu pods on this node which is not good enough. If one of 8 gpus is taken by a pod, we also want to move it to other gpu nodes.

Reply:
I would also be fine with 1.0 (always try to pack GPU as tightly as possible). It feels that with GPU you always want to either avoid restarts or minimize number used and the sort of in-between approach we use for cpu/memory (0.5) makes less sense. But I'm ok to leave this default if you think this actually makes sense.

Reply:
I would assume that PDBs are not created with ML jobs on regular basis. At least I wouldn't do this. 1.0 is extremely aggressive towards pods that finish a bit later than other pods. They will be constantly interrupted, with big chance of starting from the very beginning. 
I would go for something small - like 0.25 or, maybe, in extreme 0. 
1.0 is IMHO a wrong choice.

Reply:
0.25 inclusive means 1/4 gpu will be considered over threshold which will not trigger scale down. Most of large GPU machines comes with 4 or 8 gpus, I think it's reasonable to try to scale it down if there're 1 or 3 gpus used. 0.5 will work for both cases.